src/odata-to-abstract-sql.ts

```diff
@@ -219,7 +219,7 @@ export const rewriteBinds = (
 	existingBinds.push(...definition.extraBinds);
 };
 
-const isBindReference = (maybeBind: {
+export const isBindReference = (maybeBind: {
 	[key: string]: unknown;
 }): maybeBind is BindReference => {
```

Comment:
Do you need to also export `BindReference`? I remember some odd behaviour with implicitly/indirectly exported types on TS.

Reply:
I don't need to because it's directly available from the odata-parser. The needing to export a type is when generating declaration files (which we are) and it's nested weirdly from another module, I'm not sure the precise trigger but typescript will fail to generate the declaration file/fail the build if changes are necessary